app.py

```diff
@@ -83,6 +83,16 @@ def category_or_char(char):
             (char,),
         )
 
+    # Redirect single characters to their names
+    if len(char) == 1:
+        db.execute("SELECT name, ordinal FROM chars WHERE ordinal = ?;", (ord(char),))
+        try:
+            name, char_ord = db.fetchone()
+        except Exception:
+            pass
+        else:
+            return redirect(f"/{name}")
```

Comment:
Should we use `url_for` here instead of a hard-coded path?

Reply:
Yes! I couldn't remember the name of that one and got lazy. 😅